src/Sparrow.Server/LowMemory/MemoryInformation.cs

```diff
@@ -636,6 +656,26 @@ private static void SetMemoryRecords(long availableRamInBytes)
             HighLastFiveMinutes = highLastFiveMinutes;
             LowLastFiveMinutes = lowLastFiveMinutes;
         }
+
+        public static bool IsHighDirtyMemory(long minimumAllowedUseInBytes, int percentageFromPhysicalMem, out string details)
+        {
+            details = null;
+            var totalScratchMemory = GetTotalScratchAllocatedMemory();
+            
+            if (totalScratchMemory <= (TotalPhysicalMemory.GetValue(SizeUnit.Bytes) * percentageFromPhysicalMem) / 100)
```

Comment:
I don't like the separate checks.
We should error if _either_ percentage is too high or amount is too high.
Right now we error on amount only if percentage is too high